classes/help.php

```diff
@@ -71,64 +70,64 @@ function print_page() {
+									<h3><?php _e( 'Displaying Your Map', 'simplemap' ); // XSS ok. ?></h3>
 
 									<div class="inside" style="padding: 0 10px 10px 10px;">
 
 										<div class="table">
 											<table class="form-table">
 
 												<tr>
-													<td><?php _e( 'To show your map on any post or page, insert the shortcode in the body:', 'SimpleMap' ); // XSS ok. ?>
+													<td><?php _e( 'To show your map on any post or page, insert the shortcode in the body:', 'simplemap' ); // XSS ok. ?>
 														<code
 															style="font-size: 1.2em; background: #ffffe0;">[simplemap]</code>
 													</td>
 												</tr>
 
 												<tr>
-													<td><?php _e( 'If you want only certain categories or tags to show on a map, insert shortcode like this, where the numbers are replaced with the ID numbers of your desired categories and tags:', 'SimpleMap' ); // XSS ok. ?>
+													<td><?php _e( 'If you want only certain categories or tags to show on a map, insert shortcode like this, where the numbers are replaced with the ID numbers of your desired categories and tags:', 'simplemap' ); // XSS ok. ?>
 														<code style="font-size: 1.2em; background: #ffffe0;">[simplemap
 															categories=2,5,14 tags=3,6,15]</code></td>
 												</tr>
 
 												<tr>
-													<td><?php _e( 'If you want to hide the category or tag filters on the search form, insert shortcode like this:', 'SimpleMap' ); // XSS ok. ?>
+													<td><?php _e( 'If you want to hide the category or tag filters on the search form, insert shortcode like this:', 'simplemap' ); // XSS ok. ?>
 														<code style="font-size: 1.2em; background: #ffffe0;">[simplemap
 															show_categories_filter=false show_tags_filter=false]</code>
 													</td>
 												</tr>
 
 												<tr>
-													<td><?php _e( 'If you want to hide the map, insert shortcode like this:', 'SimpleMap' ); // XSS ok. ?>
+													<td><?php _e( 'If you want to hide the map, insert shortcode like this:', 'simplemap' ); // XSS ok. ?>
 														<code style="font-size: 1.2em; background: #ffffe0;">[simplemap
 															hide_map=true]</code></td>
 												</tr>
 
 												<tr>
-													<td><?php _e( 'If you want to hide the list of results, insert shortcode like this:', 'SimpleMap' ); // XSS ok. ?>
+													<td><?php _e( 'If you want to hide the list of results, insert shortcode like this:', 'simplemap' ); // XSS ok. ?>
 														<code style="font-size: 1.2em; background: #ffffe0;">[simplemap
 															hide_list=true]</code></td>
 												</tr>
 
 												<tr>
-													<td><?php _e( 'If you want to override the default lat / lng for a specific map, insert shortcode like this:', 'SimpleMap' ); // XSS ok. ?>
+													<td><?php _e( 'If you want to override the default lat / lng for a specific map, insert shortcode like this:', 'simplemap' ); // XSS ok. ?>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 191 characters

<!-- Comment by [SideCI](https://sideci.com) -->